examples/src/main/java/org/apache/spark/examples/streaming/JavaDirectKafkaWordCount.java

```diff
@@ -77,6 +93,9 @@ public static void main(String[] args) throws Exception {
     kafkaParams.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
     kafkaParams.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
     kafkaParams.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+    if (kerberosOn){
+      kafkaParams.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, "SASL_PLAINTEXT");
```

Comment:
Would be better to use the Kafka constant.

Reply:
Do you means `security.protocol` instead of 
 `CommonClientConfigs.SECURITY_PROTOCOL_CONFIG`?

Reply:
> Would be better to use the Kafka constant.

Do you means use `security.protocol1 instead of
`CommonClientConfigs.SECURITY_PROTOCOL_CONFIG`?

Reply:
SASL_PLAINTEXT string has a constant in Kafka.